### DISCUSSION TITLE
Support for Intel ARC dGPU's with OpenVINO

### DISCUSSION THREAD

**Discussion by architag21** [roflcoopter/viseron#860] [Ideas]
HI @roflcoopter, do you plan on enabling support for Arc dGPU with Openvino. As I can see currently, support is based on OpenCV’s DNN API. Any thoughts. If support is there already and I am missing something do let me know. Thanks.

**Comment by roflcoopter** [comment]
Hi! Atm i sadly dont have any time for this.

It is also a challenge to implement support for hardware that i dont own, and i dont make near enough to warrant buying one sadly.

If you find some good guide on how to do it and it doesnt seem too complicated then maybe i could take a stab.
Otherwise it would have to be contributed by someone

**Reply by architag21** [reply]
We can provide remote access to the device with dGPU if you are interested and can spend in some cycles. Do let me know. 
Also with the current OpenVINO implementation, have you tried pipeline scalability for Viseron. Did you look into Intel's dlStreamer as a reference perhaps ?

**Reply by roflcoopter** [reply]
Sorry for the very late reply, swamped with other work atm.

Remote access would be great! Will circle back to this when v3 has been fully released.



> Also with the current OpenVINO implementation, have you tried pipeline scalability for Viseron. Did you look into Intel's dlStreamer as a reference perhaps ?

This is not something i am familiar with, do you have any resource where i could read more?
